schemas/context/viewability-details.example.1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xdm:adUnitDepth": 0,
+  "xdm:viewportHeight": 1250,
+  "xdm:viewportWidth": 1600,
+  "xdm:adHeight": 250,
+  "xdm:adWidth": 300,
+  "xdm:playerVolume": 85,
+  "xdm:measurementEligible": true,
+  "xdm:implementationDetails" : {
```

Comment:
Paging @cdegroot-adobe 

This is the first time that we are using the `implementationdetails` schema. Should we have one `xdm:implementations` array for the whole experience event, or should we allow scattered pointers to `implementationdetails` in specific places?